client/components/dashboard/dashboard-config-directive.html

```diff
@@ -131,11 +131,37 @@
                  dashboardSvc.DEFAULT_TABLE_PARTITION).label }} (default)"
               ng-model="ngModel.table_partition">
         <md-option ng-repeat="option in dashboardSvc.TABLE_PARTITIONS"
-                ng-value="{{ option.partition }}"
+                ng-value="option.partition"
                 title="{{ option.tooltip }}">{{ option.label }}</md-option>
       </md-select>
     </div>
   </div>
 
+  <div class="pk-sidebar-item">
+    <div class="pk-sidebar-item-label">optimize CURRENT_TIMESTAMP</div>
```

Comment:
This must have gotten lost in a bad rebase.  There was a tooltip here (per your last suggestion); will re-add it.